style.css

```diff
@@ -133,343 +110,146 @@ section {
   font-weight: 500;
-  font-size: 12px;
-  line-height: 16px;
-  color: #6070ff;
-  margin-top: 15px;
-  text-decoration: none;
-}
-
-.about-myself-container button {
-  margin-left: 20px;
-  margin-top: 10px;
-  padding-right: 15px;
-  padding-top: 3px;
-  padding-bottom: 5px;
-  border: 1px solid #7f8cff;
-  border-radius: 5px;
-  background: #fff;
-}
-
-.form-case form button {
-  margin-left: 26px;
-  padding: 10px;
-  outline: none;
-  border: none;
-  border-radius: 5px;
-}
-
-.section-two .background button {
-  margin-top: 20px;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 7px;
-  border: 2px solid #dfe1e6;
-  border-radius: 10px;
-  gap: 12px;
-  background: #fff;
-}
-
-/* section two 2 */
-
-.section-two-2 {
-  height: 552px;
-  background-image: none;
-  width: 327px;
-}
-
-.section-two-2 .background {
-  border-radius: 10px;
-  box-shadow: 3px 3px 8px #888;
-  margin-left: 20px;
-  margin-right: 20px;
-  height: 520px;
-  padding-left: 10px;
-  padding-top: 15px;
-  background: #fff;
-  margin-top: -150px;
-}
-
-.section-two-2 .background div img {
-  width: 263px;
-  height: 220px;
-}
-
-.section-two-2 .background h2 {
-  font-style: normal;
-  font-weight: 700;
-  font-size: 32px;
-  line-height: 30px;
-  margin-top: 5px;
-  margin-bottom: 5px;
-  color: #172b4d;
-}
-
-.section-two-2 .background ul {
-  display: flex;
-}
-
-.section-two-2 .background ul li {
-  list-style-type: none;
-  padding-right: 5px;
-  font-style: normal;
-  font-weight: 600;
-  font-size: 13px;
-  line-height: 16px;
-  color: #172b4d;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
 }
 
-.section-two-2 .background p {
-  margin-top: 15px;
-  font-style: normal;
+.case1 .case2 p {
   font-weight: 400;
   font-size: 15px;
   line-height: 24px;
   color: #172b4d;
+  margin-bottom: 20px;
 }
 
-.section-two-2 .background .box {
-  display: flex;
-}
-
-.section-two-2 .background .box div a {
-  margin-right: 15px;
-  background: #ebebff;
-  border-radius: 8px;
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 4px 12px;
-  font-style: normal;
+.case1 .case2 ul li a {
+  text-decoration: none;
+  padding: 5px 10px;
+  background-color: #ebebff;
+  margin-right: 10px;
+  border-radius: 10px;
   font-weight: 500;
   font-size: 12px;
   line-height: 16px;
   color: #6070ff;
-  margin-top: 15px;
-  text-decoration: none;
 }
 
-.section-two-2 .background button {
-  margin-top: 20px;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 7px;
-  border: 2px solid #dfe1e6;
+.about-section button {
+  padding: 10px;
   border-radius: 10px;
-  gap: 12px;
-  background: #fff;
+  border: 1px solid #396df2;
 }
 
-/* the about area */
-.about-myself-container {
-  width: 375px;
-  height: 850px;
-  left: 375px;
-  top: 4769px;
-  background: #fff;
+.case1 .case2 button {
+  border: 1px solid #6070ff;
+  padding: 10px;
+  border-radius: 10px;
+  margin-top: 15px;
 }
```

Comment:
- [ ] Kindly make sure your button background color is the same as the one on the provided Figma design. :wink: 

   - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/199299493-6a4e93a1-a193-41ea-8d4c-3990ea1d6f5f.png)

  - ### Figma design
    ![image](https://user-images.githubusercontent.com/87197412/199299691-dffecaec-67a9-475a-955e-a43a0a1d56d3.png)